buzzmoon-frontend/buzzmoon-vite/src/Components/CompeteView/CompeteView.jsx

```diff
@@ -28,32 +54,89 @@ export default function CompeteView() {
     fetchGames();
   }, []);
 
-  const screenType = useBreakpointValue({base:"mobile", md:"desktop"});
-
-  return (
+  const screenType = useBreakpointValue({base:"small", md:"medium", lg:"large"});
 
-      <Routes>
-        <Route path="/"
-          element={
-            <Center my={{base:'4', md:'20'}}>
-              {screenType === "mobile" ?
-                <VStack>
-                  <GameList heading={"Play Now"} games={availableGames}></GameList>
-                  <GameList heading={"Games Played"} games={gamesPlayed}></GameList>
-                </VStack> :
-                <Flex w={{base:'90%', md:'70%'}} direction={'row'} justify={'space-between'} wrap={'wrap'}>
-                <GameList heading={"Play Now"} games={availableGames}></GameList>
-                <GameList heading={"Games Played"} games={gamesPlayed}></GameList>
-              </Flex>}
-
-            </Center>}>
-        </Route>
-        <Route path="/:gameID" element={<GameSplashPage />}></Route>
-        <Route path="/:gameID/play/:resultKey" element={<Game />}></Route>
+  if(screenType === "small"){
```

Comment:
Seems like there's a code repetition pattern introduced by the search bar. If you moved the search bar to a different place would it benefit from removing this code repetition just because of screen size?